src/Http/Controllers/Conversations/Messages/MessageController.php

```diff
@@ -2,12 +2,33 @@
 
 namespace OwowAgency\Gossip\Http\Controllers\Conversations\Messages;
 
+use Illuminate\Http\JsonResponse;
+use OwowAgency\Gossip\Models\Message;
+use OwowAgency\Gossip\Models\Conversation;
 use OwowAgency\Gossip\Http\Controllers\Controller;
 
 class MessageController extends Controller
 {
-    public function __invoke()
+    /**
+     * Paginate the messages of the given conversation.
+     *
+     * @param  int  $conversationId
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function __invoke(int $conversationId): JsonResponse
     {
-        // TODO: Implement __invoke() method.
+        $conversation = Conversation::findOrFail($conversationId);
```

Comment:
Why no [implicit binding](https://laravel.com/docs/8.x/routing#implicit-binding)?

Reply:
The model is going to be stored into a configuration value anytime soon. But this might have the same solution related to [this comment](https://github.com/owowagency/laravel-gossip/pull/5#discussion_r539100389).